excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/validation/SpreadsheetValidatorTest.java

```diff
@@ -14,160 +12,144 @@
 
-    @Test()
+    @Test
     public void shouldPass_whenMaxLengthNotConfigured() {
         String columnName = "nonExistingColumnName";
         int rowNumber = 6;
         try {
             validator.validate("sheet", columnName,
-                "TestComplexAddressBookCaseTestComplexAddressBookCaseInvalidExceedingMaxLengthValue", rowNumber);
+                    "TestComplexAddressBookCaseTestComplexAddressBookCaseInvalidExceedingMaxLengthValue", rowNumber);
         } catch (InvalidImportException ex) {
             assertThat(ex.getMessage(), is(
-                "Error processing sheet \"sheet\": Invalid columnName " + columnName + " at rowNumber " + rowNumber));
+                    "Error processing sheet \"sheet\": Invalid columnName " + columnName + " at rowNumber "
+                    + rowNumber));
             throw ex;
         }
     }
 
-    @Test(expected = InvalidImportException.class)
+    @Test
     public void shouldFail_whenBlankSheetName() {
-        try {
+        InvalidImportException ex = assertThrows(InvalidImportException.class, () -> {
             validator.validate("sheet", new DefinitionSheet(), Collections.emptyList());
-        } catch (InvalidImportException ex) {
-            assertThat(ex.getMessage(), is(
+        });
+        assertThat(ex.getMessage(), is(
                 "Error processing sheet \"sheet\": Invalid Case Definition sheet - "
-                    + "no Definition name found in Cell A1"));
-            throw ex;
-        }
+                + "no Definition name found in Cell A1"));
     }
 
-    @Test(expected = InvalidImportException.class)
+    @Test
     public void shouldFail_whenEmptyHeader() {
         final DefinitionSheet definitionSheet = new DefinitionSheet();
         definitionSheet.setName("name");
 
-        try {
+        InvalidImportException ex = assertThrows(InvalidImportException.class, () -> {
             validator.validate("sheet", definitionSheet, Collections.emptyList());
-        } catch (InvalidImportException ex) {
-            assertThat(ex.getMessage(), is(
+        });
+        assertThat(ex.getMessage(), is(
                 "Error processing sheet \"sheet\": Invalid Case Definition sheet - "
-                    + "no Definition data attribute headers found"));
-            throw ex;
-        }
+                + "no Definition data attribute headers found"));
     }
 
-    @Test(expected = Test.None.class)
+    @Test
     public void shouldValidate_WithHeadings() {
         final DefinitionSheet definitionSheet = new DefinitionSheet();
         definitionSheet.setName("name");
 
         // This test case ould fail if an exception is thrown; nothing else to assert.
         validator.validate("sheet", definitionSheet, Arrays.asList("N G I T B"));
 
-        // Could have used Latest AssertJ to assert no exception is thrown but it is not in POM
+        // Could have used Latest AssertJ to assert no exception is thrown but it is not
+        // in POM
```

Comment:
Looks like this could be moved back to the previous line as it doesn't appear to be exceeding the max number of characters on a line.